Public/Src/Cache/ContentStore/Distributed/NuCache/RocksDbContentLocationDatabase.cs

```diff
@@ -194,9 +194,15 @@ private BoolResult Load(OperationContext context, StoreSlot activeSlot, bool cle
 
                 Tracer.Info(context, $"Creating rocksdb store at '{storeLocation}'.");
 
-                var possibleStore = KeyValueStoreAccessor.Open(storeLocation,
-                    additionalColumns: new[] { nameof(Columns.ClusterState), nameof(Columns.Metadata) },
-                    rotateLogs: true,
+                var possibleStore = KeyValueStoreAccessor.Open(

```

Comment:
Comments please on what each of these does.

Reply:
Added inline comments. RocksDb options are commented in the documentation for the arguments object.

---
In reply to: [349360402](https://github.com/microsoft/BuildXL/pull/1257#discussion_r349360402) [](ancestors = 349360402)